tests/containers/push_container_image_to_acr.pm

```diff
@@ -9,16 +9,20 @@
 
 use Mojo::Base 'publiccloud::basetest';
 use testapi;
-use containers::urls 'get_suse_container_urls';
+use containers::urls qw(get_suse_container_urls get_urls_from_var);
 
 sub run {
     my ($self, $args) = @_;
 
     $self->select_serial_terminal;
 
     my $provider = $self->provider_factory(service => 'ACR');
-    my ($untested_images, $released_images) = get_suse_container_urls();
-    my $image = $untested_images->[0];
+
+    my $image;
+    unless ($image = get_urls_from_var()->[0]) {
```

Comment:
Same as previous. This `unless` block is hard to understand and the name `get_urls_from_var` requires explicit knowledge of the function to get an idea what happens.

Reply:
I added the variable as a parameter and also added a comment. Can you check again?